.github/workflows/pr-verify.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: pr-verify
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
Can we enable on linux also and then we can simplify the Jenkins file and have it do just the release ?